warehouse/templates/manage/account.html

```diff
@@ -276,41 +276,46 @@ <h2 class="sub-title">Change password</h2>
 
   <hr>
 
-  <section>
+  <section id="two-factor">
     <h2 class="sub-title">Two Factor Authentication</h2>
-    {% if user.totp_provisioned %}
-    <section id="delete-two-factor">
-      <p>
-        TOTP is currently <strong>enabled</strong>.
-      </p>
-      {{ form_error_anchor(delete_totp_form) }}
-      <form method="POST">
-        <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
-        {{ form_errors(delete_totp_form) }}
-        <div class="form-group">
-          {{ delete_totp_form.password(placeholder="Your current password", required="required", class_="form-group__input", data_target="password.password")}}
-          {{ field_errors(delete_totp_form.password) }}
-        </div>
-        <div>
-          <input value="Disable" type="submit" class="button button--primary">
-        </div>
-      </form>
-    </section>
-    {% else %}
-    <section id="add-two-factor">
-      <p>
-        TOTP is currently <strong>disabled</strong>.
-      </p>
-      {{ form_error_anchor(add_totp_form) }}
-      <form method="POST">
-        <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
-        {{ form_errors(add_totp_form) }}
-        <div>
-          <input value="Enable" type="submit" class="button button--primary">
-        </div>
-      </form>
-    </section>
-    {% endif %}
+    <p>Two factor authentication adds an additional layer of security to your account. <a href="/help#2fa">Learn more</a></p>
+
+    <table class="table table--light table--2fa">
+      <thead>
+        <tr>
+          <th>Two factor method</th>
+          <th colspan="2">Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Authentication app (TOTP)</td>
+          <td>{% if user.totp_provisioned %}
+              <span class="badge badge--success">Enabled</span>
+              {% else %}
+              <span class="badge badge--danger">Disabled</span>
+              {% endif %}
+          </td>
+          <td class="table__action">
+            {% if user.totp_provisioned %}
+            <a href="#disable-totp" class="button button--primary">Disable</a>
+            {{ confirm_modal(
```

Comment:
This tag and below are causing a linter assertion:

```
/home/travis/virtualenv/python3.6.6/bin/html_lint.py --printfilename --disable=optional_tag,names,protocol,extra_whitespace,concerns_separation `find ./warehouse/templates -path ./warehouse/templates/legacy -prune -o -name '*.html' -print`
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.6/bin/html_lint.py", line 66, in <module>
    sys.exit(html_linter.main(options))
  File "/home/travis/virtualenv/python3.6.6/lib/python3.6/site-packages/html_linter.py", line 1094, in main
    clean_html = template_remover.clean(io.open(filename).read())
  File "/home/travis/virtualenv/python3.6.6/lib/python3.6/site-packages/template_remover.py", line 258, in clean
    return _TemplateRemover(html_content, pattern).get_clean_content()
  File "/home/travis/virtualenv/python3.6.6/lib/python3.6/site-packages/template_remover.py", line 199, in get_clean_content
    assert False, 'Echo tags should be in just one line.'
AssertionError: Echo tags should be in just one line.
make: *** [lint] Error 1
The command "make lint BINDIR="$(dirname $(which python))"" exited with 2.
```

Maybe this is worth a behavior change? I find these easier to read than their single line equivalents.

Or perhaps https://github.com/deezer/template-remover isn't capable of removing multi-line tags, hence the assertion?

Reply:
You could do the following to collapse the width somewhat.

```
{% set title="Disable 2FA by TOTP application" %}
{% set confirm_button_label="Disable TOTP application" %}
{{ confirm_modal(title=title, confirm_name="Username", confirm_string=user.username, confirm_button_label=confirm_button_label, slug="disable-top", action=None, warning=False, confirm_string_in_title=False) }}
```

Reply:
I ended up going with a slight variation on @ewdurbin's suggestion.